Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -220,36 +231,55 @@ done < netdev.log
 			# Either VM hang or VM resume needs longer time.
 			throw "VM resume did not finish, the latest state was $state"
 		}
-		#Verify the VM status after power on event
-		$vmStatus = Get-AzVM -Name $vmName -ResourceGroupName $rgName -Status
-		if ($vmStatus.Statuses[1].DisplayStatus -eq "VM running") {
-			Write-LogInfo "$($vmStatus.Statuses[1].DisplayStatus): Verified successfully VM status is running after resuming"
-		} else {
-			Write-LogErr "$($vmStatus.Statuses[1].DisplayStatus): Could not find the VM status after resuming"
-			throw "Can not identify VM status after resuming"
-		}
 
-		# Verify the kernel panic, call trace or fatal error
-		$calltrace_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -iE '(call trace|fatal error)'" -ignoreLinuxExitCode:$true
+			# Verify the VM status after VM is accessible.
```

Comment:
indent

Reply:
Fixed in ae66ef26caf39800d996c15cf19eed2b44ecfc9f